packages/third-parties/src/ThirdPartyScriptEmbed.tsx

```diff
@@ -15,6 +17,15 @@ export default function ThirdPartyScriptEmbed({
   children,
   dataNtpc = '',
 }: ScriptEmbed) {
+  useEffect(() => {
+    // Useful for feature detection and measurement
+    performance.mark('mark_use_counter', {
+      detail: {
+        feature: `next-third-parties-${dataNtpc}`,
```

Comment:
if `dataNtpc` is not provided the feature will be `next-third-parties-`. Should we provide a more descriptive placeholder value? Also if the implementor doesn't provide a `dataNtpc` value should we even do the attribution? If it's actually required then maybe we should warn if Dev if missing.

Reply:
That's a good point. I don't think we need to fire attribution if we don't receive `dataNtpc`, since it won't be helpful